compile.sh

```diff
@@ -93,23 +92,23 @@ if [ "$MISSING_PACKAGES" != "" ]; then
 
 	# apt-get guide.
 	apt-get --help > /dev/null 2> /dev/null && \
-	missingDepsExit "sudo apt-get install$MISSING_PACKAGES"
+	missingDepsExit "sudo apt-get install $MISSING_PACKAGES"
```

Comment:
In other words, now this is what's printed when e.g. make is missing:
`sudo apt-get install<space><space>make`